ConsoleApp/Program.cs

```diff
@@ -0,0 +1,136 @@
+            formatted = "3. Sort some random string:\n {0}.\n Get all symbols in alphabetical order:\n{1}\n.";
+            Console.WriteLine(formatted,str4, SortCharInString(str4));
+
+            string str5 ="1.some test string with some words 1 and 2";
+
+            
+
+            Console.WriteLine($"4.Analyse random string:\n {str5}.\n Found next dublicated symbols:");
+            Duplicate(str5);
+
+            static void Duplicate(string str)
+            {
+                bool triple;
+                int j = 0;
+                char [] array= new char[str.Length-1];
+                for (int i = 0; i < str.Length - 1; i++)
+                {
+                    //Console.WriteLine(str[i]);
+
+                    if ((str.IndexOf(str[i],i+1)!=-1)&& (Char.IsWhiteSpace(str[i])==false))
+                    {
+                        //Console.WriteLine($"duplicate: {str[i]} on positon {i} and {str.IndexOf(str[i], i + 1)}");
+                        triple = false;
+                        for (int h = 0; h < array.Length - 1; h++)
+                        {
+                            if (str[i] == array[h])
+                                triple = true;
+                        }
+                        if (triple == false)
+                        {
+                            array[j] = str[i];
+                            j++;
+                        }
+                    }
+                }
+                //string str = string.Join(", ", array);
+                Array.Resize(ref array, j);
+                PrintArray(array);
+                //Console.WriteLine("array of duplicate");
+                
+
+            }
+            static void PrintArray(char[] arr)
+            {
+                string str=string.Join(", ", arr);
+                str =str.TrimEnd(',');
+                Console.WriteLine(str);
+            }
+
+
+            static string SortCharInString(string str)
+            { 
+            char[] array = new char [str.Length];
+                for (int i = 0; i < array.Length; i++)
+                {
+                    array[i] = Char.ToLower(str[i]);
+                }
+                //Array.Sort(array); =)
+                for (int j = str.Length; j > 0; j--)
+                {
+                    char tempChar;
+                    for (int i = 0; i < j-1; i++)
+                    {
+                        if (array[i] > array[i + 1])
+                        {
+                            tempChar = array[i];
+                            array[i] = array[i + 1];
+                            array[i + 1] = tempChar;
+                        }
+                    }
+                }
+
+                return new string(array);
+            }
+
+            static bool Compare(string first, string second)
+            {
+                if (first.Length != second.Length)
+                    return false;
+                else
+                {
```

Comment:
you can avoid `else` statement and simply write the code the outside `if else`

Reply:
thx